Tabela.html

```diff
@@ -1,42 +0,0 @@
-<!doctype html>
-<html>
-	<head>
-		<meta charset="UTF-8">
-		<title> tabela</title>
-		<link rel = "stylesheet" href = "style.css" type = "text/css" />
-		
-</head>
-	
-	
-	<body>
-
-		<div id="tabela">
-			<table bordercolor="black" cellspacing = "0" cellpadding = "10" border = "2">
-				<tr bgcolor = "yellow" border-top-color = "blue">
-					<td> Numer  </td>  <td> Imie  </td>  <td> Nazwisko   </td>  	<td> Ulubiony Kolor  </td>  
-				</tr>
-				<tr>
-					<td> 1 </td>  	<td> Piotr </td>  	<td> Czyż </td>  	<td> Fioletowy </td>  
-				</tr>
-				<tr class = "moja">
```

Comment:
Check this article: https://www.w3schools.com/cssref/tryit.asp?filename=trycss3_nth-child_odd_even